webpack/scenes/AlternateContentSources/MainTable/ACSTable.js

```diff
@@ -80,6 +93,19 @@ const ACSTable = () => {
       dispatch(getAlternateContentSources())));
   };
 
+  const onBulkDelete = (ids) => {
+    setDeleting(true);
+    dispatch(bulkDeleteACS({ ids }, () => {
+      setDeleting(false);
+      dispatch(getAlternateContentSources());
```

Comment:
We'd want here is a check to see if one of the selected ACS to be deleted is also expanded with URL as /details/:id cause that throws an error. You can re-purpose the handleSuccess function code in onDelete to check if acsId is in the ids list and push('/alternate_content_sources') if it is.


```
const onBulkDelete = (ids) => {
    setDeleting(true);
    dispatch(bulkDeleteACS({ ids }, () => {
      setDeleting(false);
      if (acsId && ids.has(Number(acsId))) {
        push('/alternate_content_sources');
      } else {
        dispatch(getAlternateContentSources());
      }
    }));
  };
```

Reply:
Pushed the fix!